data/json/items/melee/spears_and_polearms.json

```
@@ -348,6 +348,28 @@
     "use_action": [ "HEAT_SOLID_ITEMS" ],
     "melee_damage": { "bash": 6, "stab": 30 }
   },
+  {
+    "type": "ITEM",
+    "id": "spear_machined",
+    "name": { "str": "machined steel spear" },
+    "description": "A stout wooden pole with a cylindrical steel spearhead that was turned on a lathe.",
```

I assume you mean it has a point?
```suggestion
    "description": "A stout wooden pole with a conical steel spearhead that was turned on a lathe.",
```